src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -197,7 +199,26 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     mpu: props =>
       isAdsEnabled ? <MpuContainer {...props} slotType="mpu" /> : null,
     wsoj: props => (
-      <CpsRecommendations {...props} items={recommendationsData} />
+      // 005_brasil_recommendations_experiment
+      <OptimizelyExperiment experiment={OPTIMIZELY_CONFIG.experimentId}>
+        {variation => {
+          let unirecsHybridRecommendationData = null;
+          if (variation && variation !== 'control') {
+            unirecsHybridRecommendationData = path(
+              [OPTIMIZELY_CONFIG.variationMappings[variation]],
+              pageData,
+            );
+          }
+
+          return (
+            <CpsRecommendations
+              {...props}
+              parentColumns={gridColsMain}
+              items={unirecsHybridRecommendationData ?? recommendationsData}
+            />
+          );
+        }}
+      </OptimizelyExperiment>
```

Comment:
Can this use the `wsoj: props => <OptimizelyRecommendation pageData={pageData} {...props} />` like the ArticlePage does?